Libplanet.Tests/Net/SwarmTest.Preload.cs

```diff
@@ -1167,5 +1167,97 @@ public async Task PreloadAsyncCancellation(int cancelAfter)
                 );
             }
         }
+
+        [Fact(Timeout = Timeout)]
+        public async void GetDemandBlockHashes()
+        {
+            Swarm<DumbAction> minerSwarm = _swarms[0];
+            Swarm<DumbAction> receiverSwarm = _swarms[1];
+            Log.Logger.Information("Miner:    {0}", minerSwarm.Address);
+            Log.Logger.Information("Receiver: {0}", receiverSwarm.Address);
+
+            BlockChain<DumbAction> minerChain = _blockchains[0];
+            BlockChain<DumbAction> receiverChain = _blockchains[1];
+
+            (_, IEnumerable<Block<DumbAction>> blocks) =
+                await MakeFixtureBlocksForPreloadAsyncCancellationTest();
+
+            foreach (Block<DumbAction> block in blocks)
+            {
+                minerChain.Append(block);
+            }
+
+            minerSwarm.FindNextHashesChunkSize = 2;
+            await StartAsync(minerSwarm);
+
+            (BoundPeer, long?)[] peers =
+            {
+                ((BoundPeer)minerSwarm.AsPeer, minerChain.Count - 1),
+            };
+
+            (long, HashDigest<SHA256>)[] demands = await receiverSwarm.GetDemandBlockHashes(
+                receiverChain,
+                peers,
+                progress: null,
+                cancellationToken: CancellationToken.None
+            ).ToArrayAsync();
+
+            IEnumerable<(long, HashDigest<SHA256>)> expectedBlocks = minerChain.IterateBlocks()
+                .Where(b => b.Index >= receiverChain.Count)
+                .Select(b => (b.Index, b.Hash));
+            Assert.Equal(expectedBlocks, demands);
+        }
+
+        [Fact(Timeout = Timeout)]
+        public async void GetDemandBlockHashesDuringReorg()
```

Comment:
```suggestion
        public async Task GetDemandBlockHashesDuringReorg()
```

Reply:
Fixed in 5efe719923cce1445c58c7804bf04b6b292c620b (amended).